### PR TITLE
Add support for default input, seed, enum, file pickup, and raw output

### PR DESCRIPTION
**Pull request by katieannbaker**
Adding support for inputs: seed & enum types
Adding support for outputs: file pickup & raw/string output

### CODE REVIEW

File: nodes/io/input_output.py [hunk 1]
```diff
@@ -20,10 +20,11 @@ def INPUT_TYPES(cls):
             "required": {
                 "input_name": ("STRING", {}),
                 "input_desc": ("STRING", {}),
-                "input_type": (["STRING", "FLOAT", "INT", "BOOLEAN", "IMAGE", "MASK"],),
+                "input_type": (["STRING", "FLOAT", "INT", "BOOLEAN", "IMAGE", "MASK", "SEED", "ENUM"],),
                 "input_value": ("STRING", {"multiline": True, "dynamicPrompts": False}),
+                "user_override_required": ("BOOLEAN", {}),
```
Comment (katieannbaker) [inline nodes/io/input_output.py:25]: @WAS-PlaiLabs With a BOOLEAN type - will this show up as a checkbox on the front end and defaulted to unchecked (false)?

File: nodes/io/input_output.py [hunk 2]
```diff
@@ -139,7 +148,7 @@ def INPUT_TYPES(cls):
                 "animation_fps": ("INT", {"min": 1, "max": 60, "default": 8}),
                 "animation_quality": (["DEFAULT", "HIGH"],),
             },
-            "hidden": {"unique_id": "UNIQUE_ID"},
+            "hidden": {"unique_id": "UNIQUE_ID", "output_subdir": None},
```
Comment (katieannbaker) [inline nodes/io/input_output.py:151]: @WAS-PlaiLabs I couldn't find a lot of documentation on adding hidden attributes, and wasn't sure about what the value should be in this dict. My goal is for this field to not be something that a user can set on the front end, but I want to be able to provide a value in the middle layer to set it.

File: nodes/io/input_output.py (hunk 1 above)
Comment (WAS-PlaiLabs) [reply nodes/io/input_output.py:25]: No, all values are received via `STRING` input_value widgets. So you'd write 0 or 1, or True or False. This is then cast to bool and returned. If it's not a valid Boolean it'll resolve to False.

Comment (katieannbaker) [reply nodes/io/input_output.py:25]: I guess I'm more curious about the user experience on the front end. The middle layer is going to be the only thing interacting with this, so I can cast it as needed in that layer.

**Review by Tylermarques — APPROVED** [review]

**Pull request merged**